extensions/shared/with-custom-class-names/index.jsx

```diff
@@ -3,19 +3,23 @@
  */
 import { createHigherOrderComponent } from '@wordpress/compose';
 
+/**
+ * Internal dependencies
+ */
 import './style.scss';
 
 // Injecting the `has-warning` class into the block wrapper component gives us
```

Comment:
It seems like we need to change the comment here `has-warning`, `is-premium`, `is-interactive` do not exist anymore in this scope. Should we move the comment to where these classes actual reside `extensions/shared/register-jetpack-block.js` ❓

Reply:
I was asking myself that as well. The HOC is used in a couple of places and I wasn't sure about move that huge copy block to two different places. Technically that would be better though. Those comments get stripped out for prod anyway, so no issue to duplicate it.